internal/store/database_users.go

```diff
@@ -124,15 +124,15 @@ func (s *Store) DBUserCertificates(projectID, username string, opts *atlas.ListO
 }
 
 // CreateDBUserCertificate creates a new Atlas managed certificates for a database user.
-func (s *Store) CreateDBUserCertificate(projectID, username string, monthsUntilExpiration int) (*atlasv2.UserCert, error) {
+func (s *Store) CreateDBUserCertificate(projectID, username string, monthsUntilExpiration int) (string, error) {
 	switch s.service {
 	case config.CloudService, config.CloudGovService:
 		userCert := &atlasv2.UserCert{
 			MonthsUntilExpiration: pointer.Get(monthsUntilExpiration),
 		}
-		_, err := s.clientv2.X509AuthenticationApi.CreateDatabaseUserCertificate(s.ctx, projectID, username, userCert).Execute()
-		return userCert, err
+		cert, _, err := s.clientv2.X509AuthenticationApi.CreateDatabaseUserCertificate(s.ctx, projectID, username, userCert).Execute()
+		return cert, err
 	default:
-		return nil, fmt.Errorf("%w: %s", errUnsupportedService, s.service)
+		return "", fmt.Errorf("%w: %s", errUnsupportedService, s.service)
```

Comment:
this is the only major change beside lib bump

Reply:
Is it fine to print the cert or we need file support?

Reply:
you mean a `-o` support? feel free to open a ticket as a follow up